drivers/hmis/app/graphql/types/hmis_schema/enrollment_summary.rb

```diff
@@ -0,0 +1,45 @@
+module Types
+  class HmisSchema::EnrollmentSummary < Types::BaseObject
+    field :id, ID, null: false
+    field :entry_date, GraphQL::Types::ISO8601Date, null: false
+    field :project_name, String, null: false
+    field :project_type, Types::HmisSchema::Enums::ProjectType, null: false
+    field :project_id, String, null: false
+    field :move_in_date, GraphQL::Types::ISO8601Date, null: true
+    field :in_progress, Boolean, null: false
+    field :can_view_enrollment, Boolean, null: false
+    field :primary_key, ID, null: true
+
+    def id
+      object.enrollment_id
+    end
```

Comment:
We don't typically resolve the "HUD IDs" for enrollments. The id can just be `object.id` since the object is an enrollment.
```suggestion
```